.github/actions/validate-with-schema/action.yml

```diff
@@ -39,6 +46,7 @@ runs:
         files: |
           ${{ inputs.data-location }}
         json_schema: schema/${{ inputs.schema-location }}/${{ inputs.schema-version }}.json
+        json_schema_version: ${{ inputs.meta-schema-version }}
```

Comment:
It's a bit of a shame this action doesn't get the metadata schema from `"$schema": "https://json-schema.org/draft/2020-12/schema"` defined in the schema.. Feel free to just resolve this

Reply:
It is a shame :/ but what can ya do...unless we make our own :D